Terraform.sublime-syntax

```diff
@@ -303,7 +303,7 @@ contexts:
 
   # Terraform "import" statements
   #
-  # https://www.terraform.io/docs/import/usage.html
+  # https://www.terraform.io/docs/cli/import/usage.html
```

Comment:
Generally "import" is not part of the language, it's just a regular CLI command, like any other, such as `terraform apply` so I'm not sure what the motivation to introduce it to the grammar was in the first place, but this is admittedly not related to your PR here.

Reply:
Right! I just wanted to be consistent with the changes. This should perhaps be discussed in a separate issue, but I personally do not use Sublime.